pkg/minikube/constants/constants.go

```diff
@@ -31,7 +31,7 @@ const (
 	MiniShiftHomeEnv             = "MINISHIFT_HOME" // Environment variable used to change the Minishift home directory
 	VersionPrefix                = "v"
 	MinOpenshiftSupportedVersion = "v1.4.1"
-	DefaultMemory                = 2048
+	DefaultMemory                = "2g"
```

Comment:
I'd rather use '2GB' and then change '20g' to '20GB' as well. The unit is really GB, right?